BugReporter/Info/AssemblyInfo.cs

```diff
@@ -1,12 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------------------------------
-// <copyright file="AssemblyInfo.cs" company="NBug Project">
-//   Copyright (c) 2011 - 2013 Teoman Soygul. Licensed under MIT license.
-// </copyright>
-// --------------------------------------------------------------------------------------------------------------------
-
-namespace BugReporter.Info
-{
-    public class AssemblyInfo
```

Comment:
Are this and the following two `...Info` classes accessed using reflection in the BugReporter, @RussKie?

Reply:
I don't think so... I did a quick search for these types, and I didn't see any uses. Let's nuke 'em :)